.ci/jenkins_job_build.sh

```diff
@@ -99,7 +99,9 @@ ${GOPATH}/src/${tests_repo}/.ci/install_go.sh -p
 # Run the static analysis tools
```

Comment:
In the commit message s/checker things/checker thinks/

Reply:
Fixed.